### ISSUE TITLE
High api usage

### ISSUE THREAD

**Issue by snowe2010** [gitify-app/gitify#986]
### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### 📝 Description

Gitify is using an insane amount of api calls, causing me to hit my api usage every hour. Closing gitify solves the issue. Just opening Gitify uses up over a hundred api calls. (104 in my most recent test)

### 🪜 Steps To Reproduce

run 

```  gh api \
        -H "Accept: application/vnd.github+json" \
        -H "X-GitHub-Api-Version: 2022-11-28" \
        /rate_limit
```

to check your usage. 

Wait a minute, run again and your usage will be several _hundred_ api calls higher. 

### Gitify Version

5.2.0

### Operating System

macOS

### GitHub Account

GitHub.com

### 📸 Screenshots

![image](https://github.com/gitify-app/gitify/assets/1534535/482b57ce-d998-42c2-b3aa-474c465be6e3)
 then after opening gitify
![image](https://github.com/gitify-app/gitify/assets/1534535/771dd0a7-9880-4081-8cc2-958c9669e4e0)

**Comment by afonsojramos** [comment]
We fetch notifications every minute, but your issue something else. Can you try disabling GitHub-like state colors? That is a potential high usage setting 😬

**Comment by snowe2010** [comment]
I turned github like state colors off and it seems to have reduced the usage. I went from 24 to 128 on startup, then disabled the colors, but then 1 minute later the usage jumped to 230, but finally seems to have slowed down. Setting might have taken a minute to go into effect. What is this setting? I can't find it on the website, faq, or readme.

**Comment by setchy** [comment]
Thanks for reporting @snowe2010

While we look into optimizing our API calls + caching, I have raised two PRs to help with the reported issues above
- #1005
- #1006

**Comment by snowe2010** [comment]
Thank you very much!

**Comment by setchy** [comment]
These additionally should also help reduce the symptoms
* https://github.com/gitify-app/gitify/pull/1012
* https://github.com/gitify-app/gitify/pull/1014

**Comment by setchy** [comment]
@snowe2010 - all the above are now released in 5.3.0

**Comment by snowe2010** [comment]
thank you very much! y'all are fast!
